bigtable/tests/unit/test_row.py

```diff
@@ -330,35 +335,56 @@ def test_delete_cells_with_string_columns(self):
                 column_qualifier=column2_bytes,
             ),
         )
-        self.assertEqual(row._pb_mutations, [expected_pb1, expected_pb2])
+        self.assertEqual(row.row_mutations.mutations_entry.mutations[0],
+                         expected_pb1)
+        self.assertEqual(row.row_mutations.mutations_entry.mutations[1],
+                         expected_pb2)
+
+    def _make_responses(self, codes):
+        import six
+        from google.cloud.bigtable_v2.proto.bigtable_pb2 import (
+            MutateRowsResponse)
+        from google.rpc.status_pb2 import Status
+
+        entries = [MutateRowsResponse.Entry(
+            index=i, status=Status(code=codes[i]))
+            for i in six.moves.xrange(len(codes))]
```

Comment:
This would be clearer using `enumerate`, e.g.:

```python
        entries = [MutateRowsResponse.Entry(
            index=index, status=Status(code=code))
            for index, code in enumerate(codes)]
```

Reply:
@aneepct: ping.